public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -105,6 +119,9 @@ export function AppChrome({ children }: Props) {
         </>
       )}
       {!state.chromeless && <CommandPalette />}
+      {config.featureToggles.returnToPrevious && state.returnToPrevious && shouldShowReturnToPrevious && (
```

Comment:
don't need the `state.returnToPrevious` bit as it's already contained in `shouldShowReturnToPrevious`.
tbh you could probably also move the `config.featureToggles.returnToPrevious` bit up into `shouldShowReturnToPrevious`. then this would just be:
```tsx
{shouldShowReturnToPrevious && <ReturnToPrevious ... />}
```

Reply:
Hi @ashharrison90,
I can move `config.featureToggles.returnToPrevious` into `shouldShowReturnToPrevious`, that's a good idea, but if I don't check `state.returnToPrevious` there, I get a squiggly red line on `href` and `title` saying they might be undefined.